vendors/pmx/drivers/pmx-tcr/driver.yaml

```diff
@@ -0,0 +1,3 @@
+# A reference to the corresponding driver on our private drivers repository
+protocolId:
+  - pmx:pmx-tcr-protocol:1
```

Comment:
We normally use this reference if a driver already exists on our private drivers repository. So this file alongside the folder "drivers" should be removed until adding the driver.

Reply:
What do you mean with "private drivers repository"? I removed the driver reference for now. 
In the moment it is important that our customers are able the select a PMX TCR and having the right profiles in place (LoRaWAN 1.0.4 or 1.1)